test/acceptance/testing.go

```diff
@@ -499,3 +501,16 @@ func RetryFor(timeout time.Duration, f func(c chan struct{}) error) error {
 		return ctx.Err()
 	}
 }
+
+// ExponentialBackoff returns a function that retries using
+// a back-off strategy of retrying 'n' times and doubling the
+// amount of time waited after each one.
+func ExponentialBackoff(limit time.Duration) ShouldRetryFunc {
```

Comment:
This function name and comment describe exponential backoff, but I'm pretty sure it's actually linear? We should either change the name to reflect reality, or change the impl to actually do exponential backoff. The comment "doubling every time" makes me think you intended to do exponential backoff, but before we go and change it - what has the introduction of backoff done to the flake rate of the tests so far? If the current, linear impl has been fine, we could just change the function name and comment, WDYT?

```suggestion
func LinearBackoff(limit time.Duration) ShouldRetryFunc {
```

Reply:
> what has the introduction of backoff done to the flake rate of the tests so far?

AFAIK, we experienced less regression in our nightly tests, but I guess it's something that takes longer to observe.

![image](https://user-images.githubusercontent.com/16480203/153878765-2112c805-9896-409d-bdd3-93717375d52b.png)

My bad you're right, it's linear

Reply:
Great, the overall regression is down - but I meant, do we observe _individual_ tests that have had retry logic added to them, flaking still?

Reply:
> Great, the overall regression is down - but I meant, do we observe individual tests that have had retry logic added to them, flaking still?

Some of them yeah, but less often. I looked at circleci insights and according to them we don't have any flacky test anymore. The retry policy definitely decrease regression for those tests.

![image](https://user-images.githubusercontent.com/16480203/153881760-93d4653f-cde3-467b-a832-a30fde0360d2.png)

Reply:
I got one in my mind (AWS_EIP) that has retry logic but still sometimes (last night) failed. But the rest seems to be good for now.

Reply:
nice, that's an improvement! Maybe we want to try a switch to longer timeout, maybe with exponential backoff? Let's merge this for sure though.